.github/workflows/test.yml

```diff
@@ -47,10 +47,14 @@ jobs:
         uses: actions/setup-python@v1
         with:
           python-version: "3.6"
-      - name: Install Python formatter
+      - name: Install Python Black code style formatter
         run: pip install black
-      - name: Check Python formatting
+      - name: Install Python isort formatter
+        run: pip install isort
+      - name: Check Black formatting
         run: black --check languages/python
+      - name: Check isort formatting
+        run: isort languages/python -c
```

Comment:
```suggestion
        run: isort languages/python -c  --profile black
```
Perhaps?

Reply:
That's indeed probably the better option :smile: